### PR TITLE
Added another commit with some changes to Contribution_Guidlines.md

### PR DESCRIPTION
**Pull request by Hiccup1234**
More information can be added after discussion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,12 @@
 
 ## Setup
 
-- Visit painite-tc repo at ```https://github.com/technical-council-nitt/painite-tc```
```
Comment (jaiakash) [inline README.md:9]: The 3 backticks denote code in the markdown file. That's why I added the 3 backticks for links, websites or commands.
Just now, discovered both 3 backticks and single backticks work same. :+1: :100:

**Review by jaiakash — APPROVED** [review]

**Review by jaiakash — APPROVED** [review]

**Comment by jaiakash** [comment]
Closes #11

File: README.md (hunk 1 above)
Comment (Hiccup1234) [reply README.md:9]: Yeah it just auto edits

**Review by Jaiharishan — CHANGES_REQUESTED** [review]
Looks good 🔥. Also, can you combine all separate files in one file? As basically, it's just one blog.

Give a commit after that 👍

**Review by jaiakash — APPROVED** [review]

**Pull request merged**